pyproject.toml

```diff
@@ -10,6 +10,7 @@ python = "^3.7"
 numpy = "^1.10.1"
 scipy = "^1.0.0"
 pandas = "^1.0.0"
+soundfile = "^0.10.0"
```

Comment:
I have trouble installing this with `poetry install`, which results in: `SolverProblemError  Because wfdb depends on soundfile (^0.10.0) which doesn't match any versions, version solving failed.`. Running `poetry add soundfile@^0.10.0` results in `SoundFile = "^0.10.0"` being added

https://pypi.org/project/SoundFile/